### ISSUE TITLE
Should ASE be a core dependency?

### ISSUE THREAD

**Issue by jwa7** [lab-cosmo/equisolve#29]
In equisolve/utils/convert.py, ase is used in the `ase_to_tensormap()` fxn. If not already installed, a `ModuleNotFoundError` is raised. Should ASE be a core dependency?

**Comment by Luthaf** [comment]
I don't think it should be, equisolve does not need ase to work. But the ase import should be guarded when defining the `ase_to_tensormap` function (if ase is not available, this function will never be used anyway)

**Comment by agoscinski** [comment]
We just do it like in chemiscope and make a HAS_ASE flag, as we also will do it with for torch
